src/main/java/reposense/parser/AuthorConfigCsvParser.java

```diff
@@ -52,6 +52,10 @@ protected void processLine(List<RepoConfiguration> results, String[] elements)
         List<String> aliases = getManyValueInElement(elements, ALIAS_POSITION);
         List<String> ignoreGlobList = getManyValueInElement(elements, IGNORE_GLOB_LIST_POSITION);
 
+        if (branch.isEmpty()) {
+            branch = DEFAULT_BRANCH;
+        }
```

Comment:
Not a very neat solution here. 
Perhaps adding a `getValueInElement` overload method which sets the default value would be better.

E.g. `getValueInElement(elements, BRANCH_POSITION, DEFAULT_BRANCH);`